styles/global.css

```diff
@@ -1,35 +1,113 @@
-:root {
-    --background: rgb(224, 234, 247);
-    --header-background: white;
-    --footer-background: rgb(243, 242, 242);
-}
+/*color palette 
```

Comment:
Great job laying out your colors! there is a way to save these all as variables that you can use, [https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties](url) . checkout that link if you are interested in that kind of thing. Let me know if you have any questions!